charts/prometheus/README.md

```diff
@@ -56,6 +57,10 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 16.0
+
+Start from 16.0, prometheus moves all embedded charts out of prometheus chart umbrella back to its parent. Version 16.0.0 moves node-exporter to external promtheus-node-exporter. If you've made some manual changes, please diff the old (`nodeExporter`) and the new (`prometheus-node-exporter`) configuration section in values.yaml. Uninstalling the old chart release and then reinstalling the new chart release is recommended.
```

Comment:
I guess "uninstalling" is not the best idea, as the prometheus chart default is to use a deployment and the pvc would likely be deleted too, so you would loose all data.

So i would try too give non destructive upgrade instructions instead. Best thin would be to try it out once and describe any special steps which might have to be done.